site/content/docs/5.0/getting-started/introduction.md

```diff
@@ -59,8 +63,11 @@ Be sure to have your pages set up with the latest design and development standar
   <body>
     <h1>Hello, world!</h1>
 
-    <!-- Optional JavaScript -->
-    <!-- Popper.js first, then Bootstrap JS -->
+    <!-- Optional JavaScript; choose one of the two! -->
+    <!-- Bootstrap Bundle with Popper.js -->
+    <script src="{{< param "cdn.js_bundle" >}}" integrity="{{< param "cdn.js_bundle_hash" >}}" crossorigin="anonymous"></script>
+
+    <!-- Or Popper.js first, then Bootstrap JS -->
```

Comment:
Not sure if these should be commented out?

Reply:
I still think one of the two should be commented out since people tend to copy the blocks as is.

Reply:
Agreed